### ISSUE TITLE
Is It works on EF Core

### ISSUE THREAD

**Issue by Andrewdorien** [MikaelEliasson/EntityFramework.Utilities#129]
I'm using EF Core 2.6 right now and have generic repository for my Entities in project.
I can't use IDbSet instead of DbSet<TEntitty> because I have to totally change my repository...
Is it any way to use this utility library on EF "Core".

**Comment by RudeySH** [comment]
No, this library is only suitable for Entity Framework 6. Entity Framework Core (1.0) came out later and wasn't backwards compatible, and this library was never rewritten to support it.
If you want to, you can use Entity Framework 6 (non-Core) in a .NET Core 3.1 project (or .NET Standard 2.1), but I recommend against doing this if you're already using Entity Framework Core.
